aibolit/patterns/loop_outsider/loop_outsider.py

```
@@ -2,6 +2,10 @@
 # SPDX-License-Identifier: MIT
 from typing import List
 from aibolit.types_decl import LineNumber
+from javalang import parse
+
+from aibolit.ast_framework import AST, ASTNodeType
+from aibolit.ast_framework.ast_node import ASTNode
```

Ruff (0.11.9)

`aibolit.ast_framework.ast_node.ASTNode` imported but unused

Remove unused import: `aibolit.ast_framework.ast_node.ASTNode`

(F401)